### ISSUE TITLE
Unable to use status.Conditions due to package missing/moved

### ISSUE THREAD

**Issue by RobEarl** [operator-framework/operator-sdk#3991]
#### What did you do?

Following the instructions here - https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/

I imported `github.com/operator-framework/operator-sdk/pkg/status` in my `_types.go` file.

#### What did you expect to see?

I expected to be able to use status.Conditions.

#### What did you see instead? Under which circumstances?

```
github.com/operator-framework/operator-sdk/pkg/status: module github.com/operator-framework/operator-sdk@latest found (v1.0.1), but does not contain package github.com/operator-framework/operator-sdk/pkg/status
```

#### Environment

**Operator type:**

/language go

**Kubernetes cluster type:**

`$ operator-sdk version`
`operator-sdk version: "v1.0.0", commit: "d7d5e0cd6cf5468bb66e0849f08fda5bf557f4fa", kubernetes version: "v1.18.2", go version: "go1.13.11 linux/amd64", GOOS: "linux", GOARCH: "amd64"`

`$ go version` (if language is Go)

`go version go1.13.6 linux/amd64`

`$ kubectl version`

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.2", GitCommit:"f5743093fd1c663cb0cbc89748f730662345d44d", GitTreeState:"clean", BuildDate:"2020-09-16T13:41:02Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.3+3107688", GitCommit:"3107688", GitTreeState:"clean", BuildDate:"2020-07-17T21:45:20Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"linux/amd64"}
```

#### Possible Solution

I tracked the package down in operator-lib - https://github.com/operator-framework/operator-lib/blob/main/status/conditions.go

Possibly just a documentation change.

**Comment by estroz** [comment]
Yep this is just a documentation issue. Feel free to make a PR!

/kind documentation

**Comment by joelanford** [comment]
We _just_ merged a PR that removes the `status` package in `operator-lib`. The upstream community now has official conditions types and helpers. See https://github.com/operator-framework/operator-lib/issues/10#issuecomment-705087512

/assign @varshaprasad96

**Comment by varshaprasad96** [comment]
Yes, the [PR](https://github.com/operator-framework/operator-lib/pull/35) for removing status package from operator-lib just got merged. Will create a follow-up to update doc in SDK.

**Comment by estroz** [comment]
Should we update the v1.0.x docs to use `operator-lib/status` like we intended, or also point those docs to upstream conditions?

**Comment by joelanford** [comment]
I'd vote for pointing to upstream probably.

**Comment by RobEarl** [comment]
@estroz will do, thanks for confirming!

**Comment by estroz** [comment]
Closed by #4008

**Comment by estroz** [comment]
On second thought this cannot be in v1.0.x because upstream `metav1.Condition` wasn't released until k8s v1.19 and v1.0.x (and v1.1.x) are on k8s v1.18. Also the `go/v2` plugin is still on v1.18 so we probably can't have this in our master docs either.

**Comment by varshaprasad96** [comment]
Closing this issue, as the [doc](https://master.sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/) has been updated for all versions post v1.0 to reflect this change.
